### PR TITLE
accessing the rest API

### PR DESCRIPTION
**Pull request by nadinequinr3**
- Add info about calling the REST API
- Made general update that any links in left-hand menu that point outside the Docs Portal should open in a new tab

Preview here: https://nadine.preview.docs.r3.com/en/platform/corda/5.0/reference/rest-api.html

### CODE REVIEW

File: content/en/platform/corda/5.0/reference/rest-api/accessing.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+To access and invoke the REST API:
+
+1. If the REST API has not been exposed externally to the cluster via a load balancer, forward the port by running the following command in a terminal window, replacing `<NAMESPACE>` with the Kubernetes namespace that Corda is installed in:
+
+   ```sh
+   kubectl port-forward -n <NAMESPACE> deploy/corda-rest-worker 8888
+   ```
+
+   If you did not explicitly specify the username for the initial admin user at install time, the default is `admin`.
+If you did not explicitly specify the password for the initial admin user at install time, you can retrieve it using the following command:
```
Comment (yarivus) [inline content/en/platform/corda/5.0/reference/rest-api/accessing.md:23]: is the different indent intentional ?

**Review by yarivus — COMMENTED** [review]
see comment

**Review by yarivus — COMMENTED** [review]
see prev comment

File: content/en/platform/corda/5.0/reference/rest-api/accessing.md (hunk 1 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0/reference/rest-api/accessing.md]: Fixed. Thanks

File: content/en/platform/corda/5.0/reference/rest-api/accessing.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+   REST_API_USER=admin
+   REST_API_PASSWORD=$(kubectl get secret -n <NAMESPACE> corda-initial-admin-user -o go-template="{{ .data.password | base64decode }}")
+   curl -u $REST_API_USER:$REST_API_PASSWORD -k $REST_API_URL/api/v1/hello
+   ```
+
+   {{< note >}}
+   The REST API is protected by a self-signed certificate.
+   {{< /note >}}
+
+You can access the Swagger documentation for the REST API at the path `/api/v1/swagger`. For example, when using port forwarding the documentation is available at `<REST_API_URL>/api/v1/swagger`.
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/reference/rest-api/accessing.md:41]:
```suggestion
You can access the Swagger documentation for the REST API at the path `/api/v1/swagger`. For example, when using port forwarding, the documentation is available at `<REST_API_URL>/api/v1/swagger`.
```

File: content/en/platform/corda/5.0/reference/rest-api/accessing.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+You can also view the REST API documentation [here](../../rest-api/C5_OpenAPI.html).
```
Comment (paulmoloneyr3) [inline content/en/platform/corda/5.0/reference/rest-api/accessing.md:43]: Should this link work?

**Review by yarivus — APPROVED** [review]
LGTM

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/5.0/reference/rest-api/accessing.md (hunk 3 above)
Comment (nadinequinr3) [reply content/en/platform/corda/5.0/reference/rest-api/accessing.md]: Good spot - thanks! Fixed and rebuilding preview now 👍

**Review by paulmoloneyr3 — APPROVED** [review]

**Review by yarivus — COMMENTED** [review]
thx @nadinequinr3

**Pull request merged**